app/javascript/dashboard/routes/dashboard/commands/commandbar.vue

```diff
@@ -56,6 +57,16 @@ export default {
     setCommandbarData() {
       this.$refs.ninjakeys.data = this.hotKeys;
     },
+    onSelected(item) {
+      const {
+        detail: { action: { title = null, section = null } = {} } = {},
+      } = item;
+      this.$track(GENERAL_EVENTS.COMMAND_BAR, {
+        section,
```

Comment:
Added a section here because sometimes the action involves dynamic values.
<img width="791" alt="CleanShot 2023-08-02 at 12 57 57@2x" src="https://github.com/chatwoot/chatwoot/assets/12408980/d12252e9-6b69-4329-9041-bade08a34272">